### PR TITLE
<ASVisibility> Fixes to visibility depth behavior on iOS 7.

### PR DESCRIPTION
**Pull request by garrettmoon**

### CODE REVIEW

**Comment by maicki** [comment]
LGTM

File: AsyncDisplayKit/ASTabBarController.m [hunk 1]
```diff
@@ -39,6 +39,12 @@ - (void)visibilityDepthDidChange
 
 - (NSInteger)visibilityDepthOfChildViewController:(UIViewController *)childViewController
 {
+  NSUInteger viewControllerIndex = [self.viewControllers indexOfObject:childViewController];
+  if (viewControllerIndex == NSNotFound) {
+    //If childViewController is not actually a child, return NSNotFound which is also a really large number.
+    return NSIntegerMax;
```
Comment (maicki) [inline AsyncDisplayKit/ASTabBarController.m]: Should that be NSNotFound? or maybe adjust the comment

Comment (garrettmoon) [inline AsyncDisplayKit/ASTabBarController.m]: It should! Was switching between the two. Will fix.

**Comment by hannahmbanana** [comment]
Hmmm...this Travis failure is mysterious. I'll re-run it overnight and see if it is a build server issue.

**Pull request merged**